packages/labs/nextjs/src/index.ts

```diff
@@ -6,16 +6,29 @@
 
 import type {NextConfig} from 'next';
 
-// TODO(augustjk) Consider adding option for adding template shadowroot polyfill
-// automatically
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface LitSsrPluginOptions {}
+interface LitSsrPluginOptions {
+  includeDSDPolyfill?: boolean;
```

Comment:
Could this be documented, so when someone is investigating the options bag type, they get documentation for this config.

Reply:
Yes, good call. I'd like to do that when we settle on a name.
Apparently I'm supposed to do strict camel casing https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined so `includeDsdPolyfill`.

 @michaelwarren1106 suggested on discord maybe spelling it out like `polyfillDeclarativeShadowDom` which uses "polyfill" as a verb rather than doing "include". Naming is very hard.

Reply:
`addDeclarativeShadowDomPolyfill` isn't that bad really, IMO. I personally rather config be very explicit for future readers who didn't set it up in the first place.